### PR TITLE
update the html

### PR DESCRIPTION
**Pull request by MarinaAlmeida20**
added some code in there.

### CODE REVIEW

**Review by hermes-almeida — COMMENTED** [review]
need add some css.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>First Repo</title>
+  </head>
+  <body>
+    <h1>Hello, This is my first repo.</h1>
```
Comment (hermes-almeida) [inline index.html:10]: add more H1 here

Comment (MarinaAlmeida20) [reply index.html:11]: please check if you like this, instead h1 was added h2

File: style.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+body {
```
Comment (MarinaAlmeida20) [inline style.css:1]: added a stylesheet to the project

**Pull request closed without merging**